### PR TITLE
refactor(alert): change alert default icon size to 24px

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Refactor

#### What is expected?

Change default size of icon in Alert from 32px to 24px.

## Relevant logs and/or screenshots

Before:

Screenshot 2022-12-02 at 11 10 19

After:

Screenshot 2022-12-02 at 11 10 23

### CODE REVIEW

File: src/components/Alert/index.tsx [hunk 1]
```diff
@@ -117,20 +85,21 @@ type AlertProps = {
...
-    />
+  <StyledStackContainer
+    direction="row"
+    alignItems="center"
+    justifyContent="flex-start"
+    type={type}
+    className={className}
+    gap={2}
+  >
+    <Icon name={icon || typesDefaultIcons[type]} size={24} aria-hidden="true" />
```
Comment (chambo-e) [inline src/components/Alert/index.tsx:102]: Should we really allow customisation of icon here ? 🤔

Comment (matthprost) [reply src/components/Alert/index.tsx:102]: @chambo-e you're right it is not a good behavior, I will remove it

Comment (matthprost) [reply src/components/Alert/index.tsx:102]: @chambo-e my bad I've read too fast, there might be use case where icon will be different especially for `info` variant

Comment (matthprost) [reply src/components/Alert/index.tsx:102]: example: Screenshot 2022-12-02 at 16 19 41

Comment (matthprost) [reply src/components/Alert/index.tsx:102]: We need to define this with design, I think it will come later on with a bigger refactor for this component.

Comment (chambo-e) [reply src/components/Alert/index.tsx:102]: Gotcha 

(omg that icon is awful)

**Review by ModuloM — APPROVED** [review]

**Pull request merged**